lib/config/options/index.ts

```diff
@@ -1633,6 +1633,21 @@ const options: RenovateOptions[] = [
     default: 'dependency {{depName}}',
     cli: false,
   },
+  {
+    name: 'prettyNewVersion',
+    description: 'Filed exposed to templating to append v to newVersion',
+    type: 'string',
+    default:
+      '{{#if (isFirstCharacterV newVersion)}}{{{newVersion}}}{{else}}v{{{newVersion}}}{{/if}}',
```

Comment:
```suggestion
      '{{prettifyVersion newVersion}}',
```
would it be better to incapsulate this logic in the handlerbars helper it self? e.g. {{prettifyVersion newVersion}} like suggested?